eng/bomgenerator/generateAzureSDKBOM.ps1

```diff
@@ -43,6 +43,10 @@ if(! (Test-Path $InputDir)) {
   New-Item -Path $PSScriptRoot -Name "inputDir" -ItemType "directory"
```

Comment:
Similar comment: `"inputDir"` or `$InputDir`?

Reply:
It is intentionally "inputdir" here